lib/game.rb

```diff
@@ -0,0 +1,91 @@
+
+class Game
+  attr_accessor :board, :player_1, :player_2, :winner, :current_player, :move
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]
+  ]
+
+
+  def initialize(player_1=Player::Human.new("X"), player_2=Player::Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+    @turn_count = 0
+
+  end
+  
+
+  def current_player
+    (board.turn_count + 1) % 2 == 0 ? @player_2 : @player_1
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def won?
+    @winning_array = WIN_COMBINATIONS.detect{|a| 
+      a.all?{|p| board.cells[p] == "X" } || a.all?{|p| board.cells[p] == "O" }
+      } 
+    @winning_array ? true : false
```

Comment:
Good job storing the winning move. This makes `winner` more efficient. However, you could rely on ruby's implicit return value instead of using a ternary operator (if `detect` fails, it will return nil). If you want to ensure a `true` or `false` value, you could also use the double bang operator, e.g.  `!!@winning_array`.